geopandas/geodataframe.py

```diff
@@ -133,6 +133,8 @@ class GeoDataFrame(GeoPandasBase, DataFrame):
 
     def __init__(self, data=None, *args, geometry=None, crs=None, **kwargs):
         with compat.ignore_shapely2_warnings():
+            if kwargs.get("copy") is None and type(data) == DataFrame:
```

Comment:
```suggestion
            if (
                kwargs.get("copy") is None
                and isinstance(data, DataFrame)
                and not isinstance(data, GeoDataFrame)
            ):
```

I was actually suggesting this. In this case we make a copy of a DataFrame but not of a GeoDataFrame, therefore being consistent with what pandas is doing (I think).

Reply:
✅